utils/snappy-move-results/common.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Get OpenShift cluster details
+export cluster_name=$(oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}')
+export masters=$(oc get nodes -l node-role.kubernetes.io/master --no-headers=true | wc -l)
+export workers=$(oc get nodes -l node-role.kubernetes.io/worker --no-headers=true | wc -l)
+export infra=$(oc get nodes -l node-role.kubernetes.io/infra --no-headers=true | wc -l)
+export platform=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}')
+export cluster_version=$(oc get clusterversion | grep -o [0-9.]* | head -1)
+export network_type=$(oc get network cluster  -o jsonpath='{.status.networkType}' | tr '[:upper:]' '[:lower:]')
+export folder_date_time=$(TZ=UTC date +"%Y-%m-%d_%I:%M_%p")
+export SNAPPY_USER_FOLDER=${SNAPPY_USER_FOLDER:=perf-ci}
+
+if [[ -n $RUNID ]];then 
+    export runid=$RUNID-
+fi
+
+#Function to store the run id, snappy path and other cluster details on elasticsearch
+store_on_elastic()
+{
+    if [[ -n $RUNID ]];then 
+        export ES_SERVER="https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443"
```

Comment:
we already have an ES_SERVER variable from scale-ci-deploy or e2ebenchmarking workloads. I'm afraid this will override the value already set.

pick the value of ES_SERVER from the env, and use this default dev instance in case no ES_SERVER var is defined.

Reply:
i'll just change the name of the var , cause the index for snappy exists on this es instance .